lib/xcribe.ex

```diff
@@ -24,4 +101,23 @@ defmodule Xcribe do
 
     [worker(Xcribe.Recorder, [])]
   end
+
+  @doc false
+  def case do
```

Comment:
https://hexdocs.pm/elixir/library-guidelines.html#avoid-use-when-an-import-is-enough

That is actually an antipattern.

Why don't just let the user:

```
use Xcribe.Information
```

And move the document macro to this file here, so he can just import it directly:
```
import Xcribe
```

I think makes sense in some situations have the `using` but right here seems to me that we are adding another level of indirection without necessity.

WDYT @Danielwsx64 ?

Reply:
I thought would be simpler if we expose just `Xcribe` to user interact. But you have a point.

So we back to use `Xcribe.Information` and I removed the namespace `Helpers` from `Xcribe.Helpers.Document`. So now on `ConnCase` we need just import `Xcribe.Document`.

Tks so much for the review @thiamsantos